crates/nq-tokio-network/src/lib.rs

```diff
@@ -120,19 +119,19 @@ impl Network for TokioNetwork {
 pub struct TokioNetworkInner {
     connections: Arc<ConnectionManager>,
     time: Arc<dyn Time>,
-    shutdown: Arc<ShutdownHandle>,
+    shutdown: Arc<CancellationToken>,
```

Comment:
The [internal implementation](https://docs.rs/tokio-util/latest/src/tokio_util/sync/cancellation_token.rs.html#56) of a `CancellationToken` is already wrapped in an `Arc`, so we can do away with this extra `Arc`.